File: eCommerce/UI/ecommerce_mobile/lib/model/limit.dart
import 'package:ecommerce_mobile/model/kategorija_transakcije.dart';
import 'package:ecommerce_mobile/model/user.dart';
import 'package:json_annotation/json_annotation.dart';

part 'limit.g.dart';

@JsonSerializable()
class Limit {
  int? limitId;
  int? iznos;

  int? korisnikId;
  User? korisnik;
  
  int? kategorijaTransakcijeId;
  KategorijaTransakcije? kategorijaTransakcije;


  Limit({
    this.limitId,
    this.korisnikId,
    this.korisnik,
    this.iznos,
    this.kategorijaTransakcijeId,
    this.kategorijaTransakcije,
  });

  factory Limit.fromJson(Map<String, dynamic> json) => _$LimitFromJson(json);

  Map<String, dynamic> toJson() => _$LimitToJson(this);
} 

File: eCommerce/eCommerce.Services/Database/Limit.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCommerce.Services.Database
{
    public class Limit
    {
        public int LimitId { get; set; }
        public int Iznos { get; set; }
        public int KorisnikId { get; set; }
        public User Korisnik { get; set; }
        public int KategorijaTransakcijeId { get; set; }
        public KategorijaTransakcije KategorijaTransakcije { get; set; }
    }
}
